src/views/exportJarFile.ts

```diff
@@ -0,0 +1,335 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { EOL, platform } from "os";
+import { basename, extname, join } from "path";
+import { CancellationToken, commands, Extension, extensions, MessageItem, MessageOptions,
+         ProgressLocation, QuickInputButtons, Uri, window, workspace, WorkspaceFolder } from "vscode";
+
+import { isStandardServerReady } from "../extension";
+import { Jdtls } from "../java/jdtls";
+import { INodeData } from "../java/nodeData";
+
```

Comment:
`ts` is different from `java`, please do not make empty lines between import statement